Document how to disable ingress access logs (#518)
nais/doc: docs/nais-application/ingress.md

* `AND message: GET*` (replace `GET` with the HTTP method you want to filter out)

### Disable _your_ access logs

!!! note "Not reccomended"
    Running without access logs is not reccomended and will limit your ability to audit or debug connection problems with your application.

In some cases (such as legcay applications that are using personally identifiable information as URL parameters) you might want to disable access logs for a given application. This can be done by setting the following annotation in your nais yaml:

```yaml
apiVersion: nais.io/v1alpha1
kind: Application
metadata:
  name: myapplication
  namespace: myteam
  annotations:
    nginx.ingress.kubernetes.io/enable-access-log: "false"
spec:
  ...
```

> To keep personal identifiable information out of access logs use POST data instead or switch to user identifiers that are unique to your application or domain.

Reviewer: 👏

> To keep personal identifiable information out of access logs use POST data instead or switch to unique identifiers such as universally unique identifier (UUID).

Author: ```suggestion
To keep personal identifiable information out of access logs use POST data instead or switch to user identifiers that are unique to your application or domain.
```

> 

Reviewer 2: "Switch to unique identifiers" ble litt rart i mine øyne, fnr er vel (i stort) også unike. Hva med noe sånt som "switch to using `request_id` or some other identifier that is not (directly) connected to the person"?

Author: Hvis det benyttes in UUID som er unik per tjeneste eller domene så er jo det noe som ikke uten videre kan knyttes opp til personens navn. Personnummer er ikke problemet i seg selv men det at det er globalt og lett kan knyttes til annen informasjon er en del av problemet.

Reviewer 2: Jeg er med. Syns bare "unique identifier" bør utdypes litt :-) Hva med "an identifier only known to your service/system"?

Author: Enig!
